services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from datetime import datetime, date
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: date,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,

```

Comment:
The `movie_show_time` parameter should be of type `datetime`, not `date`, to match the `show_time` field in the `MovieSession` model, which is a `DateTimeField`.